packages/quantic/force-app/main/default/lwc/quanticResultDate/quanticResultDate.js

```diff
@@ -0,0 +1,85 @@
+import LOCALE from '@salesforce/i18n/locale';
+import {getBueno} from 'c/quanticHeadlessLoader';
+import {LightningElement, api} from 'lwc';
+
+/** @typedef {import("coveo").Result} Result */
+
+/**
+ * The `QuanticResultDate` component displays a given result date field value.
+ * @category Result Template
+ * @example
+ * <c-quantic-result-date result={result} label="Date" field="date"></c-quantic-result-date>
+ */
+export default class QuanticResultDate extends LightningElement {
+  /**
+   * The [result item](https://docs.coveo.com/en/headless/latest/reference/search/controllers/result-list/#result) to use.
+   * @api
+   * @type {Result}
+   */
+  @api result;
+  /**
+   * The date field whose values you want to display.
+   * @api
+   * @type {string}
+   */
+  @api field;
+  /**
+   * (Optional) The label to display.
+   * @api
+   * @type {string}
+   * @defaultValue `none`
+   */
+  @api label;
+  /**
+   * A function used to format the displayed value.
+   * The default format uses the default format for your locale.
```

Comment:
I find the current wording confusing. What would you think of something like this?

> By default, the date is formatted using the browser locale.

Reply:
Haha yeah it's pretty badly phrased as it is now. I clearly hadn't reread it. I'll take your suggestion thanks